### PR TITLE
updates to documentation

### PR DESCRIPTION
**Pull request by alexarobu**

### CODE REVIEW

File: docs/UserDoc_Chap3_Viewing.rst [hunk 1]
```diff
@@ -6,19 +6,19 @@ This section will guide users and visitors to the site on viewing metadata and p
...
+record. Below the record title, the **abstract** provides a brief description of what the dataset (or service) is and what it is used for. The
+spatial **extent** of the resource can be seen in the map below, in the **Discover data** section. Access to the data itself can be gained from the options listed in the
 **Links** section.
 
 |userdoc_fig_3_1_1_DefaultViewA|
 
 **Figure 3.1.1:** Default record view
 
-Scrolling down the page, the **Use contraints** section provides information on licencing and the **Technical information** headings provide details on topic categories and keywords, maintenance frequency, coordinate systems and data formats are provided. Information on the history of a resource and how 
+Scrolling down the page, the **Use contraints** section provides information on licencing and the **Technical information** headings provide details on topic categories and keywords, maintenance frequency, coordinate systems and data formats are provided. Information on the history of a resource and how
```
Comment (archaeogeek) [inline docs/UserDoc_Chap3_Viewing.rst]: contraints -> constraints

Comment (archaeogeek) [reply docs/UserDoc_Chap3_Viewing.rst]: Scrolling down the page, the **Use constraints** section provides information on licensing.  The **Technical information** headings provide details on topic categories and keywords, maintenance frequency, coordinate systems and data formats.

File: docs/UserDoc_Chap3_Viewing.rst [hunk 2]
```diff
@@ -63,34 +63,34 @@ To add a layer from the main map page:
...
 	|userdoc_fig_3_3_2_AddtoMapRecordView|
 
-	**Figure 3.3.2:** Adding a layer to the map from the metadata record view	
+	**Figure 3.3.2:** Adding a layer to the map from the metadata record view
 
-	*Note: with respect to WFS, users may experience occasions where the layer appears to have been added to the map, yet does not display. This 
-	occurs when the Access-Control-Allow-Orgin header (CORS access policy) is missing from the remote server, and thus the request is blocked by 
+	*Note: with respect to WFS, users may experience occasions where the layer appears to have been added to the map, yet does not display. This
+	occurs when the Access-Control-Allow-Orgin header (CORS access policy) is missing from the remote server, and thus the request is blocked by
 	the browser. This is out of the control of the portal as it is dependent on the organisations' server configuration, however, an error message shall be displayed.*
```
Comment (archaeogeek) [inline docs/UserDoc_Chap3_Viewing.rst]: shall -> will

**Review by archaeogeek — COMMENTED** [review]
Minor comments but otherwise good :-)

File: docs/UserDoc_Chap3_Viewing.rst (hunk 2 above)
Comment (alexarobu) [reply docs/UserDoc_Chap3_Viewing.rst]: updated

File: docs/UserDoc_Chap3_Viewing.rst (hunk 1 above)
Comment (alexarobu) [reply docs/UserDoc_Chap3_Viewing.rst]: updated

**Pull request merged**